ios/brave-ios/Package.swift

```diff
@@ -159,7 +159,7 @@ var package = Package(
     ),
     .target(
       name: "Storage",
-      dependencies: ["Shared"],
+      dependencies: ["Shared", "DesignSystem"],
```

Comment:
This dependency doesn't really make sense, better to create an extension on `SiteType` to expose the image from a UI related target/where it's used.